README.md

```diff
@@ -509,7 +509,9 @@ function showEmployeeList(employees) {
 
 **[⬆ back to top](#table-of-contents)**
 
-### Set default objects with Object.assign
+### Set default objects with Spread Operator
+
+When using the same name for your properties in an Object, the second property will overwrite the first.
```

Comment:
This is re-stated in the comment at the bottom where it says `config now equals`. I prefer to keep the explanation there as opposed to a note here.

Reply:
I have no objection to relocate the explanation, but I don't recommend removing it.